src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,83 @@
+package controller;
+
+import domain.*;
+import view.InputView;
+import view.OutputView;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class AttendanceController {
+    private final static int START_DATE_INDEX = 0;
```

Comment:
modifier 순서를 지켜주세요 🙂 

```suggestion
    private static final int START_DATE_INDEX = 0;
```

참고 : https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
7ee6ac1